Description: Removes the use of ssl.PROTOCOL_SSLv3
 We don't have ssl.PROTOCOL_SSLv3 support in Debian anymore, so this patch
 removes it until this is fixed upstream.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-12-10

Index: designate/designate/openstack/common/sslutils.py
===================================================================
--- designate.orig/designate/openstack/common/sslutils.py	2014-12-24 14:44:25.000000000 +0800
+++ designate/designate/openstack/common/sslutils.py	2014-12-24 14:58:02.000000000 +0800
@@ -83,8 +83,7 @@
 
 _SSL_PROTOCOLS = {
     "tlsv1": ssl.PROTOCOL_TLSv1,
-    "sslv23": ssl.PROTOCOL_SSLv23,
-    "sslv3": ssl.PROTOCOL_SSLv3
+    "sslv23": ssl.PROTOCOL_SSLv23
 }
 
 try:
